Sources/Views/PricingLanding.swift

```diff
@@ -659,8 +692,36 @@ private let featuredTeams = [
   )
 ]
 
-private func footer(currentUser: User?) -> [Node] {
-  guard currentUser == nil else { return [] }
+private func footer(
+  allEpisodeCount: AllEpisodeCount,
+  currentUser: User?,
+  subscriberState: SubscriberState
+  ) -> [Node] {
+
+  guard !subscriberState.isActive else { return [] }
+
+  let title = currentUser == nil
+    ? "Get started with our Free plan"
+    : "Get started with our Personal plan"
+
+  let subtitle = currentUser == nil
+    ? "Includes a free episode of your choice, plus weekly<br>updates from our newsletter."
+    : "Access all \(allEpisodeCount.rawValue) episodes on Point-Free today!"
```

Comment:
now the footer has a CTA for logged in, non-subscribers:

<img width="682" alt="image" src="https://user-images.githubusercontent.com/135203/63179690-b3383a00-c01a-11e9-8d7e-bc6d54b40343.png">